docs/components/truncate.md

```diff
@@ -0,0 +1,230 @@
+      short summary, it is possible to present the information in a more 
+      organized and readable manner, allowing the user to quickly scan 
+      and understand the main points of each item.</p>
+      <p>The truncated text can also serve as a teaser, encouraging the user to click or tap to view the full content.</p>
+  </KTruncate>
+</template>
+```
+
+### isExpanded
+
+If passed `true`, the component will be rendered expanded by default.
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate is-expanded>
+      <KBadge v-for="n in 25" :key="n">
+        Item {{ n }}
+      </KBadge>
+    </KTruncate>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KTruncate is-expanded>
+    <KBadge v-for="n in 25" :key="n">
+      Item {{ n }}
+    </KBadge>
+  </KTruncate>
+</template>
+```
+
+## Slots
+
+### default
+
+Slot for content.
+
+To ensure the best experience with the component, please pass elements of equal height in the default slot. The component will base its guess for the height of the element on the height of each individual sibling, so passing elements of equal height will result in a more accurate and consistent output.
+
+:::tip TIP
+The component is reactive to it's dimensions changes. To see it in effect, try resizing your browser window and see the behavior of the example below.
+:::
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate :rows="3">
+      <KBadge v-for="n in 30" :key="n">
+        Item {{ n }}
+      </KBadge>
+    </KTruncate>
+  </template>
+</KCard>
+
+### expand-trigger
+
+Slot for custom expand trigger element. Slot props:
+- `excessElementsCount` (type: `Number`) - Number of elements that overflow. **Note**: this slot prop is only available when not `isTextContent`
+- `expand` (type: `Function`) - Function to expand
+
+### collapse-trigger
+
+Slot for custom collapse trigger element. Slot props:
+
+- `collapse` (type: `Function`) - Function to collapse
+
+Example of using `expand-trigger` and `collapse-trigger` slots for creating custom toggle elements:
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate>
+      <KBadge v-for="n in 25" :key="n">
+        Item {{ n }}
+      </KBadge>
+      <template #expand-trigger="{ excessElementsCount, expand }">
+        <span class="custom-trigger" @click="expand">Show {{ excessElementsCount }} more</span>
+      </template>
+      <template #collapse-trigger="{ collapse }">
+        <span class="custom-trigger" @click="collapse">Show less</span>
+      </template>
+    </KTruncate>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KTruncate>
+    <KBadge v-for="n in 25" :key="n">
+      Item {{ n }}
+    </KBadge>
+    <template #expand-trigger="{ excessElementsCount, expand }">
+      <span @click="expand">Show {{ excessElementsCount }} more</span>
+    </template>
+    <template #collapse-trigger="{ collapse }">
+      <span @click="collapse">Show less</span>
+    </template>
+  </KTruncate>
+</template>
+```
```

Comment:
On the example here, the trigger IMO is too close to the items 
![image](https://user-images.githubusercontent.com/2229946/217009069-38e13879-fe18-4433-9634-d73031c5a137.png)

Is it possible to add a little extra space between?